server/src/domain/Experience.ts

```diff
@@ -0,0 +1,35 @@
+import { Column, Entity } from "typeorm";
+
+import { BaseModel } from "../infrastructure/BaseModel";
+
+export enum employmentType {
+    FullTime,
+    PartTime,
+    SelfEmployed,
+    Freelance,
+    InterShip,
+    ApprenticesShip,
+    Seasonal
+}
+
+@Entity({})
+export class Experience extends BaseModel {
+    @Column({ length: 100, /* unique:true */ }) // Como referencia, Github en un titulo de repositorio acepta como maximo 100
```

Comment:
Please remove commented code.


```suggestion
   /**
     * Based on GitHub's repository names
     * which allows up to 100 chars.
     */
    @Column({ length: 100 })
```

Reply:
@nachodevd whenever you want to document some piece of code, you must use doc comments. Doc comments are detected by most IDEs and LSPs so you will get rich DX for free.